app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -98,6 +109,77 @@ class ConversationViewModel @Inject constructor(
         }
     }
 
+    fun showDeleteMessageDialog(messageId: String) =
+        updateDialogState {
+            it.copy(deleteMessageDialogState = DeleteMessageDialogState.Visible(messageId = messageId, conversationId = conversationId!!))
+        }
+
+    fun showDeleteMessageForYourselfDialog(messageId: String) {
+        //hide deleteMessageDialog
+        updateDialogState { it.copy(deleteMessageDialogState = DeleteMessageDialogState.Hidden) }
+        //show deleteMessageForYourselfDialog
```

Comment:
I think this comment is not necessary it is clear from the method that this is happening

Reply:
Done!